src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js

```diff
@@ -22,34 +23,64 @@ class TwilioVoiceBasicCallControl extends HTMLElement {
 
     this.shadowRoot
       .querySelector('#hold')
-      .addEventListener('click', () => this.#handleHold());
+      .addEventListener('click', () => this.#handleHold(true));
     this.shadowRoot
       .querySelector('#resume')
-      .addEventListener('click', () => this.#handleResume());
+      .addEventListener('click', () => this.#handleHold(false));
+    this.shadowRoot
+      .querySelector('#mute')
+      .addEventListener('click', () => this.#handleMute(true));
+    this.shadowRoot
+      .querySelector('#unmute')
+      .addEventListener('click', () => this.#handleMute(false));
   }
 
   #handleCallMessageReceived(message) {
     const { content, messageType } = message;
     if (messageType === 'user-defined-message') {
       this.#callSid = content.callSid;
       this.#conferenceSid = content.conferenceSid;
-      this.#showButtons('hold');
+      this.#showButtons('hold', 'hold');
+      this.#showButtons('mute', 'mute');
```

Comment:
The usage of query and button id here is a bit confusing since they have the same name. How about pass the whole selector? `this.#showButtons('#hold-buttons ', 'hold');`